### PR TITLE
Updates cask-angular-focus module

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

File: modules/cask-angular-focus/focus.js [hunk 1]
```diff
@@ -19,8 +19,7 @@ function myFocusDirective ($timeout, myFocusManager) {
 
     link: function (scope, element, attrs) {
 
-      attrs.$observe('myFocus', function (newVal) {
-        // console.log('[myFocus] watching', newVal);
+      attrs.$observe('caskFocus', function (newVal) {
 
         var cleanup = myFocusManager.is.$watch(newVal, function (o) {
```
Comment (gaarf) [inline modules/cask-angular-focus/focus.js]: caskFocusManager

**Comment by gaarf** [comment]
still some renaming remains, including in comments

**Comment by ajainarayanan** [comment]
@gaarf Fixed required changes.

**Comment by gaarf** [comment]
LGTM :+1:

**Pull request merged**